src/Presentation/UiBuilder.php

```diff
@@ -46,6 +46,7 @@ private function renderTemplate( array $instancesViewModel, array $facetsViewMod
 		return $this->templateParser->processTemplate(
 			'Layout',
 			[
+				'hasFacets' => count( $facetsViewModel ) > 0,
```

Comment:
I think the intention will be more explicit as `showSidebar`. And then use that in `Layout` to skip rendering the `Sidebar` template. Instead of rendering the template, but with nothing inside. (Unless this causes a serious problem with the CSS layout right now.)

In the "All" view we will never have a sidebar. So theoretically if there was something else in the sidebar besides facets, then `hasFacets` wouldn't clearly be about (not) showing the sidebar.